lib/screens/wods/components/add_wod_form.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_form_builder/flutter_form_builder.dart';
+import 'package:provider/provider.dart';
+import 'package:intl/intl.dart';
+
+//my imports
+import 'package:kabod_app/core/presentation/constants.dart';
+import 'package:kabod_app/screens/commons/dividers.dart';
+import 'package:kabod_app/screens/commons/reusable_button.dart';
+import 'package:kabod_app/screens/commons/reusable_card.dart';
+import 'package:kabod_app/screens/home/repository/wod_repository.dart';
+import 'package:kabod_app/screens/wods/add_wod.dart';
+
+class AddWodForm extends StatelessWidget {
+  AddWodForm({
+    Key key,
+    @required GlobalKey<FormBuilderState> formKey,
+    @required this.widget,
+  })  : _formKey = formKey,
+        super(key: key);
+
+  final GlobalKey<FormBuilderState> _formKey;
+  final AddWodScreen widget;
+
+  final wodTypeOptions = [
+    'For Time',
+    'For Reps and Time',
+    'For Weight',
+    'AMRAP',
+    'Custom'
+  ];
```

Comment:
Also, `wodTypeOptions` should be in a different standalone file so you can reuse the same `wodTypeOptions` everywhere you want without duplicating it. Or else, when you want to change the Wod Types, you will have to change it in every file. You only use it twice yet but imagine if you had to change things on 100 files 😱